k256/src/arithmetic/field.rs

```diff
@@ -1,5 +1,7 @@
 //! Field arithmetic modulo p = 2^256 - 2^32 - 2^9 - 2^8 - 2^7 - 2^6 - 2^4 - 1
 
+#![allow(clippy::assign_op_pattern, clippy::op_ref)]
```

Comment:
Left all of the current arithmetic borrowing as-is in order to not affect performance as part of this PR, but it's probably worth benchmarking removing these in a followup and seeing if it affects performance

Reply:
Yeah I can fix in a follow up PR. I did this crypto-bigint

Reply:
It's really easy to fix automatically with `cargo clippy --fix`, but I just want to make sure it doesn't cause a performance regression